### DISCUSSION TITLE
1.4.0 Release?

### DISCUSSION THREAD

**Discussion by jqnatividad** [pemistahl/grex#81] [Q&A] [answered]
Hi @pemistahl ,
First off, thanks for merging my pull requests! (https://github.com/pemistahl/grex/pull/62 and https://github.com/pemistahl/grex/pull/68)!

I'm quite keen to use `grex` with the `cli` feature and take advantage of the [many improvements since the last release](https://github.com/pemistahl/grex/compare/v1.3.0...main).

I tried using 1.4.0-SNAPSHOT, but its not published on crates.io.

Was wondering when you plan to publish 1.4.0.

Thanks!

**Comment by pemistahl** [comment]
Hi @jqnatividad, I have no release date planned. It's done when it's done. I'm planning to add WASM support before releasing 1.4.0, so it may still take a while.

**Comment by pemistahl** [comment, marked as answer]
Hi @jqnatividad, I've just finally released version 1.4.0. It took a long time but now we have a stable release. Enjoy!
